### ISSUE TITLE
Mobile Map IO

### ISSUE THREAD

**Issue by werdnanoslen** [mozilla/global-sprint#15]
**[ Project Contact ]** @werdnanoslen
**[ GitHub Repo ]** https://github.com/werdnanoslen/mobile-map-io
**[ Track ]** Open Innovation
**[ Community ]** Hive Austin - wrangler: @omnignorant 
**[ Location ]** Austin, TX (CST)
### Description
A boilerplate mobile hybrid app for I/O with maps
***

## Want to contribute to this project during #mozsprint?
Join us at the [Global Sprint, June 1-2](https://mozilla.github.io/global-sprint). Leave a comment below if you're interested in contributing to this project during #mozsprint 2017!

***

## Note to the Project Lead :tada:
Congrats, @werdnanoslen! This is your official project listing for the [Mozilla Global Sprint 2017](https://mozilla.github.io/global-sprint). To confirm your registration, please complete and check off the following:

- [x] Complete [Open Leadership 101](https://mozilla.teachable.com/p/open-leadership-101)
- [x] Provide a GitHub repository for work and discussion on your project in a comment
  * new to GitHub?  Here's a [step-by-step guide on using the #mozsprint template](https://mozilla.github.io/global-sprint/project-lead/templates/)
- [x] Create a README file in your project repository. This file should help newcomers understand what your project is, why it's important, and kinds of help you're looking for.
- [x] Turn on your [Issue Tracker](https://mozilla.github.io/global-sprint/project-lead/templates/#4-create-issues) and create issues to describe each task that you need help with and how a contributor can get started on that task. [Create a label](https://mozilla.github.io/global-sprint/project-lead/templates/#5-add-a-mozsprint-label) called `mozsprint` and apply it to your issues.

#### Checklist for FEATURED Projects :clipboard:
To have your project FEATURED on [Mozilla Pulse](http://mozillapulse.org/), complete the following documentation. In past Sprints, well-documented featured projects have 5 times more contributions than other projects. Details about each item and more information about how to create them are [on our Project Requirements Page](https://mozilla.github.io/global-sprint/project-lead/project-requirements/). 

* [x] In your README, link to the [Mozilla Community Participation Guidelines](https://www.mozilla.org/en-US/about/governance/policies/participation/) or write your own code of conduct.
* [x] [Create file: LICENSE](http://choosealicense.com/) to give your project an open license, allowing for sharing and remixing.
* [x] Create file: CONTRIBUTING.md so others know how they can contribute. If you'd like, you can [remix this template](https://github.com/acabunoc/mozsprint-repo-template/blob/master/CONTRIBUTING.md)
* [x] Comment with an image to use on Mozilla Pulse for this project listing

Once all of the above is complete,
- [x] Leave a comment with the text `This is ready for Mozilla Pulse`. Your community wrangler will review this issue and post your project :balloon:

If you get stuck at any point, feel free to look at the [requirements page](https://mozilla.github.io/global-sprint/project-lead/project-requirements/) and [project templates](https://mozilla.github.io/global-sprint/project-lead/templates/) or reach out to your community wrangler, @chadsansing. We're here to help you through this process.

**Comment by werdnanoslen** [comment]
![image](https://cloud.githubusercontent.com/assets/764090/24939833/91ff288e-1f05-11e7-874c-09a1b79521d6.png)

**Comment by werdnanoslen** [comment]
This is ready for Mozilla Pulse

**Comment by chadsansing** [comment]
@zee-moz, @auremoser - what kind of community connections come to mind for this project?

**Comment by abbycabs** [comment]
Community should be Hive Austin! Flagging @omnignorant

**Comment by auremoser** [comment]
Is it particularly a mapping project? I wonder if it's worth connecting with Ushahidi and Data for Democracy. @omnignorant knows my friend Jonathon Morgan who worked at Ushahidi and would be able to provide some awesome mapping connections in the Austin area. I defer to @acabunoc for any recs since this has passed through her purview in the OLTS :).

**Comment by chadsansing** [comment]
Thanks, @auremoser -

**Comment by omnignorant** [comment]
Howdy @werdnanoslen @acabunoc @auremoser happy to help facilitate connections here to Jonathan, etc. just let me know if it's ready for that. @werdnanoslen will you be sharing this project and [Global Sprint](https://github.com/mozilla/global-sprint/) on Monday at the [Open Austin](http://open-austin.org) lightning talk meetup? @werdnanoslen I'm also wondering if the motivation for this work is to experiment with tech or to make mapping available for more diverse application. Could be an interesting Digital Inclusion angle here, but don't want to force that.

**Comment by werdnanoslen** [comment]
Yes re:Open Austin, although I will mostly be talking about how we can borrow Mozillas project management methods using my experience and MMIO as an example. The original motivation was to reduce overhead if hackathon projects, so experimentation is spot on. I have started considering it through a digital inclusion lens, but it's not inclusive of folks without any interest in learning to create mobile apps. How do you see it fitting in that way?

**Comment by omnignorant** [comment]
I was thinking that the image you use above provides a very interesting use-case in which a citizen is forced into an interaction with a corrupt officer of the state. That's a situation that many folks living on the margins face regularly, if perhaps the specifics of the interaction are different. So, if that's really an example use-case you're going for, this project might invite diverse audiences (as an "entry level issue") to provide use-cases that bring a digital mapping tool to an otherwise excluded population for creating an open-data set that maps opportunity neighborhoods or populations to facilitate or motivate more inclusive practices by directing resources. However, that might be way out of scope for the kind of participation you're looking for in this project. Your call!

**Comment by werdnanoslen** [comment]
Oh yes actually I discovered this motivation while working with some Pakistani hackers on a way to out bribery in Pakistan and make it something to comment on rather than hush the issue away. This project is like a lower level and also mobile-focused Crowdmap from Ushahidi. So yes I agree it could go in that direction. But for now I'm not sure if I can commit to that. I'd need more guidance from the OLT folks.

**Comment by omnignorant** [comment]
Sure @werdnanoslen and it sounds like an intro to Ushahidi might be a good idea too?

**Comment by abbycabs** [comment]
GitHub repo: https://github.com/werdnanoslen/mobile-map-io

**Comment by abbycabs** [comment]
Hey @werdnanoslen! can you add the label `mozsprint` to the issues you want worked on during the sprint? https://github.com/werdnanoslen/mobile-map-io/issues

Thanks so much!

**Comment by werdnanoslen** [comment]
Well there's only one issue right now (https://github.com/werdnanoslen/mobile-map-io/issues/5), since I'm most interested in OLT to reorient the project's direction. But i'll add more issues with that label as I go.

**Comment by werdnanoslen** [comment]
Hiya, who at Ushahidi should I speak with?

**Comment by omnignorant** [comment]
@werdnanoslen I sent an introduction email w/ you CC'd on Friday 4/22/17.

**Comment by werdnanoslen** [comment]
Ah my bad, must have missed it!

**Comment by stephwright** [comment]
Hi @werdnanoslen ... I was browsing projects today and noticed that your CONTRIBUTING.md file isn't linked from your README... would you mind adding that just so it's obvious to folks new to GitHub and the Global Sprint how they can contribute to your project?  Was also thinking you might want to break the issues down into specific actionable steps for folks who'd like to contribute on the day.

**Comment by werdnanoslen** [comment]
@stephwright [added link](https://github.com/werdnanoslen/mobile-map-io/commit/7fc49af2e7ef61312804ded30b9d2c5b1bf8fad9)

And as I'm not 100% sure of the direction I want to take those issues right now, I'd like to work with folks at the sprint to determine what those steps would be. That is, we might discover a problem together, and then define a goal and steps to complete that goal.

**Comment by abbycabs** [comment]
Thanks for participating in #mozsprint 2017! We're gearing up for 2018 and closing all the issues from last year. Want to bring your project to #mozsprint 2018, May 10-11? [Our Call for Projects](https://foundation.mozilla.org/opportunity/global-sprint/for-project-leads/) is open!

**Comment by abbycabs** [comment]
Updated link for the call for projects!: https://foundation.mozilla.org/opportunity/global-sprint/lead-a-project/
